src/data/courseDependenciesData.js

```diff
@@ -3,6 +3,121 @@ import {find} from 'lodash'
+          asOf: `2021-03-01`,
+        },
+        topics: [
+          'Install Gatsby and the Gatsby CLI',
+          'Programmatically create pages from your GraphQL data',
+          'Use GraphiQL to make queries based on source plugins',
+          'Create a blog post listing page with links to every post',
+          'Source and transform images',
+          'Make the site SEO-friendly',
+        ],
+        essentialQuestions: [
+          'How can I reduce the friction of publishing content?',
+          'What are the best practices for optimizing your website?',
+          'Why is it important to have a blog as a developer?',
+        ],
+        pairWithResources: [
+          {
+            title: 'Build a Video Chat App with Twilio and Gatsby',
+            byline: 'Jason Lengstorf・52m・Course',
+            path: '/courses/build-a-video-chat-app-with-twilio-and-gatsby',
+            slug: 'build-a-video-chat-app-with-twilio-and-gatsby',
+            description: `In this workshop, Jason Lengstorf will take you from an empty project folder all the way through deployment of a Twilio-powered video chat app built on Gatsby.`,
+            image:
+              'https://d2eip9sf3oo6c2.cloudfront.net/playlists/square_covers/000/432/507/full/TwilioGatsby_Final.png',
+          },
+          {
+            title:
+              'Eject create-react-app and Use Gatsby for Advanced React App Development',
+            byline: 'Khaled Garbaya・43m・Course',
+            path:
+              '/courses/eject-create-react-app-and-use-gatsby-for-advanced-react-app-development',
+            slug:
+              'eject-create-react-app-and-use-gatsby-for-advanced-react-app-development',
+            description:
+              'Create-React-App is a great tool for getting started with React, but at some point you may find yourself needing something more.',
+            image:
+              'https://d2eip9sf3oo6c2.cloudfront.net/playlists/square_covers/000/432/491/full/preview-full-EGH_cra-to-gatsby_424_2x.png',
+          },
+          {
+            title:
+              'Building a Serverless JAMStack Todo app with Netlify, Gatsby, GraphQL, and FaunaDB',
+            byline: 'Chris Biscardi・51m・Course',
+            path:
+              '/courses/building-a-serverless-jamstack-todo-app-with-netlify-gatsby-graphql-and-faunadb-53bb',
+            slug:
+              'building-a-serverless-jamstack-todo-app-with-netlify-gatsby-graphql-and-faunadb-53bb',
+            description: `This application serves as an introduction to building products with a JAMStack and Serverless approach.`,
+            image:
+              'https://d2eip9sf3oo6c2.cloudfront.net/tags/images/000/001/211/full/gatsby.png',
+          },
+        ],
+      },
+      {
+        id: '157',
+        type: 'course',
+        slug: 'build-a-blog-with-react-and-markdown-using-gatsby',
+        dependencies: {
+          gatsby: '^2.0.0',
+        },
+        freshness: {
```

Comment:
Freshness review to old Gatsby course, [Build a Blog with React and Markdown using Gatsby](https://egghead.io/courses/build-a-blog-with-react-and-markdown-using-gatsby)